### PR TITLE
add idam id and ccd id to logs

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-3703 - Print CCD & IDAM ID(?) in DN & AOS](https://tools.hmcts.net/jira/browse/DIV-3703)

### CODE REVIEW

File: test/unit/services/logger.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  describe('#accessLogger', () => {
+    it('returns access logger middleware that is excutable', () => {
+      const middleware = logger.accessLogger();
+      const res = {
+        statusCode: 200,
+        end: sinon.stub()
+      };
+      middleware(reqWithIdam, res, sinon.stub());
+      res.end();
```
Comment (hosainnet) [inline test/unit/services/logger.test.js]: Not sure if we're asserting anything here?

File: services/logger.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const nodeJsLogging = require('@hmcts/nodejs-logging');
+const { get } = require('lodash');
+
+const wrapWithUserInfo = (req, msg) => {
+  const idamId = get(req, 'idam.userDetails.id', 'unkown');
+  const caseId = get(req, 'session.case.id', 'unkown');
```
Comment (hosainnet) [inline services/logger.js]: Minor typo in the above 2 lines

File: services/caseOrchestrationService.js [hunk 3]
```diff
@@ -23,7 +23,10 @@ const methods = {
         return Object.assign(req.session, { case: response });
       })
       .catch(error => {
-        logger.error(`Trying to retrieve case from case orchestartion service: ${error}`);
+        logger.error({
+          message: logger.wrapWithUserInfo(req, 'Trying to retrieve case from case orchestartion service'),
```
Comment (hosainnet) [inline services/caseOrchestrationService.js]: Typo in orchestartion although was there before (and line 43)

File: test/unit/services/logger.test.js (hunk 1 above)
Comment (dtaylor7) [reply test/unit/services/logger.test.js]: same as respondent comment

Comment (hosainnet) [reply test/unit/services/logger.test.js]: 👍

**Pull request merged**